test/audienceCosineSim.test.ts

```diff
@@ -19,7 +19,7 @@ const makeCosineSimAudience = (
   ttl: 2592000,
   definition,
   id: 'testid',
-  name: 'cosineSimAudience',
+  // name: 'cosineSimAudience',
```

Comment:
Why are all of these values commented?

Reply:
The name is not being passed to the client anymore, so they can be deleted actually.
I'll do a quick cleanup.